web/robot/list_reordering.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import math
+import random
+
+class Point():
```

Comment:
Is this class used?

Reply:
Not with the version I sent you, originally I had assumed that the code had objects as Points with x and y properties.